exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -141,21 +168,47 @@ def __init__(
             default=None,
         )
         self._collector = None
+        self._grpc_client = None
```

Comment:
Can these attributes be private?

Reply:
Yes, there are `@property` methods which provide the client.